.github/workflows/js-and-android-test.yml

```diff
@@ -1,79 +1,75 @@
 name: Test pushes and PRs
 on:
-    push:
-    pull_request:
-        types: [assigned, opened, ready_for_review]
+  push:
+  pull_request:
+    types: [assigned, opened, ready_for_review]
 
 jobs:
-    test:
-        name: Run tests.
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v2
-            - uses: actions/setup-node@v2
-              with:
-                  node-version: 16
-                  cache: "npm"
-                  cache-dependency-path: "**/package-lock.json"
-            - name: Cache JS deps
-              uses: actions/cache@v2
-              id: cache
-              with:
-                  path: |
-                    **/node_modules
-                    ~/.cache
-                  key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}-${{ hashFiles('package-lock.json') }}
-            - name: Checks paths to rebuild Android
-              uses: dorny/paths-filter@b2feaf19c27470162a626bd6fa8438ae5b263721 #v2.10.2
-              id: changes
-              with:
-                  filters: |
-                      android:
-                      - 'feature-utils/**'
-                      - 'features/**'
-                      - 'platform/android/**'
-                      - 'platform/core/**'
-                      - 'platform/feature-api/**'
-                      js:
-                      - '**/*.js'
-                      - '**/*.jsx'
-                      - '**/*.ts'
-                      - '**/*.tsx'
-                      - '**/*.?js'
-                      - '.editorconfig'
-                      - '.eslintrc.cjs'
-                      apiAndFeatures:
-                      - 'build/**'
-                      - 'dev-utils/dummy-server'
-                      - 'features/**'
-                      - 'feature-utils/**'
-                      - 'platform/feature-api/**'
-                      - 'platform/podjs/**'
-                      swift:
-                      - 'platform/ios/**'
-            - name: Lint core and features
-              if: steps.changes.outputs.js == 'true'
-              run: ./build.js lint
-            - name: Install JS packages
-              if: steps.cache.outputs.cache-hit != 'true' && (steps.changes.outputs.apiAndFeatures == 'true' || steps.changes.outputs.js == 'true')
-              run: ./build.js offlineInstall
-            - name: Lint Swift
-              if: steps.changes.outputs.swift == 'true'
-              working-directory: platform/ios
-              run: docker run -t -v `pwd`:`pwd` -w `pwd` ghcr.io/realm/swiftlint:latest
```

Comment:
only diff is the elimination of this pass for linting swift as it's its own workflow from now on

Reply:
Right, but still we're pretty fond of the current indentation. If it's at all possible to keep it, I would appreciate that.